README.md

```diff
@@ -1,5 +1,9 @@
 # chrome-page-load-analysis
 
 Download the following files and save them in the repository folder.
- * [important_timestamps.csv](https://drive.google.com/file/d/1-S2O1FYM3pH80c2JKOoL8otkSBfdpZ_G/view?usp=sharing)
- * [per_second.csv](https://drive.google.com/file/d/1GvABkYjutU02kV0nCHUnL6VjbXQF4fIC/view?usp=sharing)
+* With Subresource Filter enabled:
+ * [important_timestamps.csv](https://drive.google.com/file/d/1GIxdatHZeMINjrEry29KERqGY0ROZ2Rt/view?usp=sharing)
```

Comment:
Hmmm, could we give these different names, with subresource filtering and without?